src/main.rs

```diff
@@ -82,21 +81,23 @@ fn setup(
     commands
         .spawn((Collider::cuboid(cube_size * 0.5, cube_size * 0.5, cube_size * 0.5), RigidBody::Dynamic))
         .insert(ColliderMassProperties::Mass(1.0))
+        .insert(Damping { linear_damping: 0.1, angular_damping: 0.1 })
         .insert(PbrBundle {
             mesh: meshes.add(Mesh::from(Cuboid::new(cube_size, cube_size, cube_size))),
             material: materials.add(cube_color),
-            transform: Transform::from_xyz(0.5, 0.5, 0.0),
+            transform: Transform::from_xyz(0.5, 100.0, 0.0),
             ..default()
         });
 
     // heavy cube (10 kg)
     commands
         .spawn((Collider::cuboid(cube_size * 0.5, cube_size * 0.5, cube_size * 0.5), RigidBody::Dynamic))
         .insert(ColliderMassProperties::Mass(10.0))
+        .insert(Damping { linear_damping: 0.02, angular_damping: 0.02 })
```

Comment:
Remove this, boxes of different masses should fall at the same acceleration and damping is (probably) not necessary for this project.